content/en/synthetics/guide/synthetic-test-monitors.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Understanding Synthetic Tests and Alerting Monitors
+kind: guide
+description: Learn about Synthetic monitors created with your Synthetic tests. 
+further_reading:
+- link: "/monitors/manage/"
+  tag: "Documentation"
+  text: "Manage Monitors"
+- link: "/synthetics/metrics/"
+  tag: "Documentation"
+  text: "Synthetic Monitoring Metrics"
+---
+
+## Overview
+
+When you create a Synthetic test, Datadog automatically creates an associated monitor. The monitor's alerting conditions in your test configuration impact how you are notified when a Synthetic test moves into an alerting state.
+
+{{< img src="synthetics/guide/synthetics_test_monitors/synthetic_test_monitor.png" alt="Synthetic test monitor" style="width:100%;">}}
+
+This guide describes how Synthetic test monitors behave, how to use a monitor's conditional variables in a Synthetic test configuration, and how to set downtimes and mute monitors.
+
+## Create a Synthetic test monitor
```

Comment:
Why doesn't this section live in https://docs.datadoghq.com/synthetics/api_tests/http_tests?tab=requestoptions#notify-your-team and in all individual test pages? Or in both test pages and this guide? Also, there does not seem to be a link between the test individual pages and this guide, is that normal?

Reply:
Good question! I was planning on publishing this guide first and doing a sweep through the individual API test pages second. We can collaborate on linking the two pieces of content together 😄

Reply:
Created DOCS-3508.